# config/aruco_logger_params.yaml
aruco_logger:
  ros__parameters:
    image_topic:     /camera/image_raw
    camera_info_topic: /camera/camera_info
    marker_size:     0.06           # (m)
    map_frame:       map
    base_frame:      base_link
    map_origin:      [-1.26, -1.03] # x, y offset (m)
    output_csv:      ~/aruco_log.csv
